pipeline-utils.groovy

```diff
@@ -7,12 +7,14 @@
 
 def define_properties(timer) {
 
-    // Set this to TRUE to disable the timer, and set DRY_RUN=true by default
-    def developmentPipeline = false;
+    // Name your test jobs something not starting with rhcos-
+    def developmentPipeline = !"${env.JOB_NAME}".startsWith("rhcos-");
```

Comment:
`!env.JOB_NAME.startsWith()` should work, no?

Reply:
Probably, but now I haven't tested it.